android/src/main/java/com/oney/WebRTCModule/WebRTCView.java

```diff
@@ -237,106 +130,6 @@ protected void onDetachedFromWindow() {
     }
 
     /**
-     * Callback fired by {@link #surfaceViewRenderer} when the resolution or
-     * rotation of the frame it renders has changed.
-     *
-     * @param videoWidth The new width of the rendered video frame.
-     * @param videoHeight The new height of the rendered video frame.
-     * @param rotation The new rotation of the rendered video frame.
-     */
-    private void onFrameResolutionChanged(
-            int videoWidth, int videoHeight,
-            int rotation) {
-        boolean changed = false;
-
-        synchronized (layoutSyncRoot) {
-            if (this.frameHeight != videoHeight) {
-                this.frameHeight = videoHeight;
-                changed = true;
-            }
-            if (this.frameRotation != rotation) {
-                this.frameRotation = rotation;
-                changed = true;
-            }
-            if (this.frameWidth != videoWidth) {
-                this.frameWidth = videoWidth;
-                changed = true;
-            }
-        }
-        if (changed) {
-            // The onFrameResolutionChanged method call executes on the
-            // surfaceViewRenderer's render Thread.
-            post(requestSurfaceViewRendererLayoutRunnable);
-        }
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    protected void onLayout(boolean changed, int l, int t, int r, int b) {
```

Comment:
@saghul, that's the second compile-time error I'm talking about in my previous comment. `WebRTCView` extends `ViewGroup` and `onLayout` is the implementation of an abstract method. I'm going to leave it for now so that you have a chance to look into it and I can switch back to my other tasks. If push comes to shove in the meantime, I may come back and look into it myself.